docs/gh_pages/docs/retrieval_chain.md

```diff
@@ -36,6 +36,7 @@ databases.
 
 1. Qdrant
 1. Pinecone
+1. Postgres(utilizing the pgvector extension)
```

Comment:
FYI, We need to mention the `pgvector` extension because there is another Postgres extension with a similar name, `pgvecto.rs`, which is also supported in Langchain (Ref: [PGVecto.rs](https://python.langchain.com/v0.2/docs/integrations/vectorstores/pgvecto_rs/)).